classes/game.js

```diff
@@ -5,30 +5,54 @@ class Game{
         this.turn = data.turn;
         this.board = new Board(data.board);
         this.me = new Snake(data.you);
+				this.test = data;
     }
     //todo:
     findFood(){
-			let foodx = request.body.board.food[0].x
-			let foody = request.body.board.food[0].y
-			
-			let locx = request.body.you.body[0].x
-			let locy = request.body.you.body[0].y
-			
-			if(locx < foodx){
-				data.move = 'right';
-			}else if(locx > foodx){
-				data.move = 'left';
-			}else if(locy < foody){
-				data.move = 'down';
-			}else if(locy > foody){
-				data.move = 'up';
+			let foodLength = Object.keys(this.test.board.food).length;
```

Comment:
`this.board.food` is already an array, so you can just do `this.board.food.length`, you don't need to wrap it an object.keys